docs/ruleset_functions/get_building_segment_skylight_roof_ratios.md

```diff
@@ -0,0 +1,67 @@
+
+## get_building_segment_skylight_roof_ratios
+
+Description: This function would determine skylight and envelope roof ratios for a building segment for residential, non-residential and semi-heated surface categories.  
+
+Inputs:
+
+  - **RMR**: The RMR that needs to determine skylight and envelope roof ratios.  
+
+Returns:
+
+- **skylight_roof_ratios_dictionary**: A dictionary that saves building segment ID with its the skylight and roof ratios for residential, non-residential and semi-heated surface categories.
+
```

Comment:
Based on the interpretation we need the following for EACH BuildingSegment.area_type_vertical_fenestration:
1. skylight to roof ratio for residential, non-residential and semiheated roofs, to verify skylight U-factor, SHGC & VT to be modeled in the baseline.
2. overall skylight to roof ratio for residentail+non-residential+semiheated roofs, to verify skylight area to be modeled in the baseline 

So I think we need to update this RDS as follows:
a.  Add another output value for #2 above
b. Since we may have several building segments that have the same area_type_vertical_fenestration, I think this funcion need to aggregate areas of the roof and skylights (for each category reported) across multiple building segments that have the same area_type_vertical_fenestratoin, and output a set of ratios for each area_type_vertical_fenestratoin present in the RMR. Does this make sense? I think otherwise the function is good.